source/stylesheets/_tab-code_signing.scss.erb

```diff
@@ -336,6 +345,7 @@
 
 					.expose {
 						@include display-flex;
+						width: auto;
```

Comment:
`width: auto` is the default, is this needed? Why?

Reply:
behave of the element was not the same without this

Reply:
That means that it is not `auto` without this. But I don't see where it is defined anything else. And if it is not defined anywhere, it is `auto`.

Reply:
here: https://github.com/bitrise-io/bitrise-workflow-editor/pull/202/files/b374a5b4f86fc505f878e854cdef3a53c30df3db#diff-5b6bc2fa079592de2108e6b49f8db44bR174